examples/comet_ml_plugin/comet_ml_plugin/comet_ml_example.py

```diff
@@ -0,0 +1,156 @@
+# %% [markdown]
+# (comet_ml_example)=
+#
+# # Comet Example
+# Comet’s machine learning platform integrates with your existing infrastructure and
+# tools so you can manage, visualize, and optimize models—from training runs to
+# production monitoring. This plugin integrates Flyte with Comet by configuring
+# links between the two platforms.
+import os.path
+
+from flytekit import (
+    ImageSpec,
+    Secret,
+    current_context,
+    task,
+    workflow,
+)
+from flytekit.types.directory import FlyteDirectory
+from flytekitplugins.comet_ml import comet_ml_login
+
+# %% [markdown]
+# First, we specify the project and workspace that we will use with Comet's platform
+# Please update `PROJECT_NAME` and `WORKSPACE` to the value associated with your account.
+# %%
+PROJECT_NAME = "flytekit-comet-ml-v1"
+WORKSPACE = "thomas-unionai"
+
+# %% [markdown]
+# W&B requires an API key to authenticate with Comet. In the above example,
+# the secret is created using
+# [Flyte's Secrets manager](https://docs.flyte.org/en/latest/user_guide/productionizing/secrets.html).
+# %%
+secret = Secret(key="comet-ml-key", group="comet-ml-group")
+
+# %% [markdown]
+# Next, we use `ImageSpec` to construct a container that contains the dependencies for this
+# task:
+# %%
+
+REGISTRY = "localhost:30000"
+image = ImageSpec(
+    name="unionai",
```

Comment:
same as below, we should use `ghcr.io/flyteorg`

Reply:
For `ImageSpec`, most users will not have push access to `ghcr.io/flyteorg`, so they will need to change the registry to run it.

Using `localhost:30000` means I am optimizing for the sandbox use case. An alternative is to use `os.getenv("REGISTRY", "localhost:30000")`, and let it be an environment variable.

Reply:
No problem, I like your idea